.drone.yml

```diff
@@ -11,6 +11,8 @@ steps:
 
 - name: build
   image: golang:1.19
+  environment:
+    CGO_ENABLED: 1
```

Comment:
CGO is enabled by default.  Is the problem that you need to statically compile due to sqlite being used by Drone?

Reply:
This can be backed out if needed